fsd_config/form_jsons/cof_r3/en/applicant-information.json

```diff
@@ -0,0 +1,99 @@
+{
+  "metadata": {},
+  "startPage": "/applicant-information",
```

Comment:
In the old forms, there was a unique id before each page path i.e `/applicant-information-abcdef` - was that redundant and not needed I take it for these?